notification-server/src/database.rs

```diff
@@ -46,15 +47,18 @@ impl PreparedStatements {
     pub async fn get_devices_from_account(
         &self,
         account_address: AccountAddress,
-    ) -> anyhow::Result<Vec<String>> {
+    ) -> anyhow::Result<Vec<(String, HashSet<Preference>)>> {
```

Comment:
Should I make a type for this?

Reply:
Yes, I think it would help make it easier to identify what is actually returned here.